R/verbose.R

```diff
@@ -1,5 +1,5 @@
 
-renv_verbose <- function() {
+renv_verbose <- function(prompt = interactive()) {
```

Comment:
Maybe my concern here is that "should we display some text" and "has the user asked for verbose output" are separate questions in my mind. Maybe we need a separate function here that collects those conditions, e.g.

```
renv_notification_required <- function(prompt) {
    prompt || renv_verbose()
}
```

or something like that?

Reply:
Maybe `renv_needs_notification()`? I do think the two concerns are linked because you always want to notify the user when the prompt is true.

Reply:
`renv_should_notify()`?

Reply:
BTW why is it `can_proceed()` but `renv_verbose()`? i.e. why does one get the `renv_` prefix?

Reply:
The rough convention I have: by default, internal functions should have names of the form `renv_<scope>_<name>`, unless they're generic enough to be useful in a variety of contexts (e.g. the stuff in `utils.R`).

`renv_verbose` probably could have a different name though.